docs/components/console/manage-clusters/cluster-backups.md

```diff
@@ -0,0 +1,45 @@
+---
+id: create-backups
+title: Create backup
+description: "If your organization works within Camunda's Enterprise plan, you can create cluster backups."
+---
+
+If your organization works within Camunda's [Enterprise](https://camunda.com/enterprise/) plan, you can create a manual and scheduled [backups](/components/concepts/backups.md) of your cluster.
+
+Only the three most recent successful backups of each type are kept, meaning you can have three manual and three scheduled backups.
+
+## Create a manual backup
+
+You can create a manual backup every five hours.
+
+To create a manual backup, take the following steps:
+
+1. Select the **Backups** tab.
+2. Click **Create manual backup**.
+
+![cluster-details](./img/cluster-detail-backups.png)
+
+3. A popup model will appear with more information about manual backups, including retention. Click **Create backup**.
+
+![cluster-details](./img/cluster-detail-backups-manual.png)
+
+:::note
+Only the three most recent successful manual backups are kept. If you already have three manual backups, the oldest manual backup is automatically removed.
+:::
+
+## Create a scheduled backup
+
+To create a scheduled backup, take the following steps:
+
+1. Select the **Backups** tab.
+2. Click **Create schedule**.
+
+![cluster-details](./img/cluster-detail-backups.png)
+
+3. Use the dropdown to choose the frequency - daily or weekly. Select time. Click **Create schedule**.
```

Comment:
Maybe:

3. Use the dropdown to schedule the frequency for backups - daily or weekly. 
4. Select the time of day you would like backups to be taken at this frequency.
5. Click **Create schedule**.

Reply:
@christinaausley since both buttons are **Create schedule**, do you think we need to add anything else to step 5? I was considering providing some feedback on this experience because of the identical button names. WDYT?

Reply:
Yeah, I wish the second button was just "Create" -- let me know if you'd like me to provide this feedback in Slack or otherwise.

Reply:
@christinaausley Please follow up on this.